functions/classes/class.User.php

```diff
@@ -1555,6 +1555,62 @@ private function block_remove_entry() {
         try { $this->Database->deleteRow("loginAttempts", "ip", $this->ip); }
         catch (Exception $e) { !$this->debugging ? : $this->Result->show("danger", $e->getMessage(), false); }
     }
+
+	
+	
+	/* @users and groups -------------------- */
+	
+	/**
+	 * From json {"2":"2","3":"1"}, get user list + perm
+	 * 
+	 */
+	public function get_user_permissions_from_json ($json) {
+		$groups = array();
+		foreach(json_decode($json, true) as $group_id => $perm) {
+			$group_details = $this->groups_parse(array($group_id));
+			
+			$tmp = array();
+			$tmp['group_id'] = $group_id;
+			$tmp['permission'] = $perm;
+			$tmp['name'] = $group_details[$group_id]['g_name'];
+			$tmp['desc'] = $group_details[$group_id]['g_desc'];
+			$tmp['members'] = $group_details[$group_id]['members'];
+			
+			$groups[] = $tmp;
+		}
+		return $groups;
+	}
+	
+	/**
+	 * Get accounts details of all IPAM admins
+	 * 
+	 */
+	public function get_ipam_admins () {
```

Comment:
Not needed, will call this->User->fetch_all_objects / $this->User->fetch_multiple_objects directly.